data-esnext.js

```diff
@@ -1543,6 +1548,26 @@ exports.tests = [
       }
     }
   ]
+},
+{
+  name: 'Error.cause property',
+  category: STAGE3,
+  significance: 'small',
+  spec: 'https://github.com/tc39/proposal-error-cause',
+  exec: function () {/*
+      var error = new Error('error', { cause: 'cause' })
+      return error.hasOwnProperty('cause') && error.cause === 'cause';
```

Comment:
we may want to test all 8 of the error constructors instead of just this 1

Reply:
done